controllers/users.js

```diff
@@ -22,14 +22,14 @@ router.get('/:id/spots', function(req, res) {
 // Create a new user
 router.post('/', function(req, res) {
   if (!req.body.hasOwnProperty('email') ||
-      !req.body.hasOwnProperty('password') ||
+      !req.body.hasOwnProperty('uid') ||
         !req.body.hasOwnProperty('firstName') ||
           !req.body.hasOwnProperty('lastName')) {
     res.status(400).json({error: "Missing expected body parameter."});
     return;
   }
 
-  User.create(req.body.email, req.body.password, req.body.firstName,
+  User.create(req.body.email, req.body.uid, req.body.firstName,
```

Comment:
To confirm: you're suggesting that we not store email in the users db entry because we can get it from the Auth? 

Will that work if they register with Facebook though? I don't think the authData object knows anything about the user's email unless they register with email/pw